package.json

```diff
@@ -178,13 +178,9 @@
           "eresource.all",
           "license.all",
           "location.all",
-          "order_format.all",
-          "order_type.all",
           "physical.all",
-          "receipt_status.all",
           "renewal.all",
-          "source.all",
-          "workflow_status.all"
+          "source.all"
```

Comment:
reporting_code.all?

Reply:
@craigmcnally do you mean we need to add this line? I believe it could be done with actual usage of this later

Reply:
I wasn't sure.  If we don't need it now adding it later is fine